examples/nidaqmx_analog_input/teststand_nidaqmx.py

```diff
@@ -49,8 +49,11 @@ def destroy_nidaqmx_tasks() -> None:
         session_management_client = SessionManagementClient(
             discovery_client=discovery_client, grpc_channel_pool=grpc_channel_pool
         )
+        # Timeout is added when destroying sessions to ensure that if a cancellation is performed
+        # for the measurement, session is closed within an appropriate timeframe.
```

Comment:
"Timeout is added" is appropriate for a PR description or changelog saying what you changed. For comments in the code, it's better to focus on the "why" of whatever you're trying to explain, and leave "what I changed" out of it.

Also, "... session is closed within an appropriate timeframe" makes it sound like the alternative is an inappropriate timeframe (e.g. a hang) but that's not true.

Proposed wording: "Use a timeout when destroying sessions in order to wait for the measurement to complete or be canceled."

Reply:
Sure. Updated the proposed change in this PR https://github.com/ni/measurementlink-python/pull/688